x/docs/internal/types/keys.go

```diff
@@ -5,8 +5,15 @@ const (
 	StoreKey     = ModuleName
 	QuerierRoute = ModuleName
 
-	MsgTypeShareDocument = "shareDocument"
+	MsgTypeShareDocument   = "shareDocument"
+	MsgTypeDocumentReceipt = "documentReceipt"
```

Comment:
I think we should rename this to something more clear such as `MsgTypeSendDocumentReceipt`